src/typeahead/typeahead-container.component.ts

```diff
@@ -100,8 +100,8 @@ export class TypeaheadContainerComponent {
     return this.parent ? this.parent.typeaheadItemTemplate : undefined;
   }
 
-  selectActiveMatch(): void {
-    this.selectMatch(this._active);
+  selectActiveMatch(e: any): void {
```

Comment:
Should have a type
Name should be descriptive
Should be optional or will cause breaking change, and will have to wait for major release to piblish